login.css

```diff
@@ -57,10 +57,10 @@ ul li ul{
 	padding: 0;
 	position: absolute;
 	top: 58px;
-	left: 130px;
 	width: 100px;
 	display: none;
 	visibility: hidden;
+	z-index: 5;
```

Comment:
08/08/2021: I add this so it can display the drop down menu when I hover on the link that contain other link. Because z-index : 5 will make the drop down to display over the the banner